WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -1271,10 +1273,11 @@ - (UIBarButtonItem *)saveBarButtonItem
 - (UIBarButtonItem *)shareBarButtonItem
 {
     if (!_shareBarButtonItem) {
-        WPButtonForNavigationBar *button = [self buttonForBarWithImageNamed:@"gridicons-share-ios"
-                                                                      frame:NavigationBarButtonRect
-                                                                     target:self
-                                                                   selector:@selector(sharePost)];
+        UIImage *image = [Gridicon iconOfType:GridiconTypeShareIOS];
```

Comment:
With the exception of the cancel and left chevron, all other bar buttons should have the size 24px X 24px.

The old images had that size built into them, but I think we should be setting the site explicitly here instead.